source/command-logging-and-monitoring/command-logging-and-monitoring.rst

```diff
@@ -0,0 +1,615 @@
+the ``command`` log component.
+
+The log messages are intended to match the information contained in the events above. Drivers MAY implement command logging
+support via an event subscriber if it is convenient to do so.
+
+The types used in the structured message definitions below are demonstrative, and drivers MAY use similar types instead so long as the information
+is present (e.g. a double instead of an integer, or a string instead of an integer if the structured logging framework does not support numeric types.) 
+
+For the string representations of messages defined below, for any optional field that is not available, drivers MAY either substitute the value with a
+language-appropriate representation of null (e.g. "null", "nil", "None") or omit the corresponding segment of the message altogether. 
+
+Drivers MUST not emit command log messages for commands issued as part of the handshake with the server, or heartbeat commands issued by server monitors. 
+
+Command Started Message
+-----------------------
+The structured format MUST contain the following key-value pairs:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1
+
+   * - Key
+     - Suggested Type
+     - Value
+
+   * - message
+     - String
+     - "Command started"
+
+   * - command
+     - String
+     - Canonical extJSON representation of the command. This document MUST be truncated appropriately according to rules defined in the 
+       `logging specification <../logging/logging.rst>`_, and MUST be replaced with an empty document "{ }" if the command is considered sensitive.
+
+   * - databaseName
+     - String
+     - The database name.
+
+   * - commandName
+     - String
+     - The command name.
+
+   * - requestId
+     - Int
+     - The driver-generated request ID.
+  
+   * - operationId
+     - Int
+     - The driver-generated operation ID. Optional; only present if the driver generated operation IDs and this command has one. 
+
+   * - driverConnectionId
+     - Int
+     - The driver's ID for the connection used for the command, as defined in the `connection monitoring and pooling specification <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`_. 
+
+   * - serverHost
+     - String
+     - The hostname or IP address for the server the command is being run on.
+
+   * - serverPort
+     - Int
+     - The port for the server the command is being run on. Optional; only present if a port was specified.
+
+   * - serverConnectionId
+     - Int
+     - The server's ID for the connection used for the command. Optional; only present for server versions 4.2+.
+
+   * - serviceId
+     - String
+     - The hex string representation of the service ID for the command. Optional; only present when the driver is in load balancer mode.
+
+The unstuctured form MUST be as follows, using the values defined in the structured format above to fill in placeholders as appropriate:
```

Comment:
I wasn't sure whether it was helpful or too prescriptive to define the exact stringified form. I do think consistency across drivers that produce strings could be helpful. but we won't actually be testing this. any thoughts?